coalib/testing/LocalBearTestHelper.py

```diff
@@ -26,6 +26,22 @@ def execute_bear(bear, *args, **kwargs):
     return list(bear_output_generator)
 
 
+def get_results(local_bear,
+                lines,
+                filename=None,
+                force_linebreaks=True,
+                create_tempfile=True,
+                tempfile_kwargs={},
+                settings={}):
+    with prepare_file(lines, filename,
+                      force_linebreaks=force_linebreaks,
+                      create_tempfile=create_tempfile,
+                      tempfile_kwargs=tempfile_kwargs) as (file, fname), \
+        execute_bear(local_bear, fname, file,
```

Comment:
see here , it is `fname, file`, so the `TestBear` should get the values in that order.